tests/include/detray/test/device/cuda/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+# Detray library, part of the ACTS project (R&D line)
+#
+# (c) 2024 CERN for the benefit of the ACTS project
+#
+# Mozilla Public License Version 2.0
+
+# C++17 support for CUDA requires CMake 3.18.
+cmake_minimum_required(VERSION 3.18)
+
+# Enable CUDA as a language.
+enable_language(CUDA)
+
+# Set the CUDA build flags.
+include(detray-compiler-options-cuda)
+
+include_directories( SYSTEM $<TARGET_PROPERTY:covfie::core,INTERFACE_INCLUDE_DIRECTORIES> )
+
+# Set up a test library, which the "new style" benchmarks and tests could use.
+file( GLOB _detray_test_headers
+   RELATIVE "${CMAKE_CURRENT_SOURCE_DIR}"
+   "../propagator_test.hpp"
+   "navigation_validation.hpp" 
+   "navigation_validation.cu")
```

Comment:
You have no globbing expressions in here. :thinking: Just list the files exactly like this, in the `add_library(...)` call.

Reply:
Done